server/src/main/java/org/elasticsearch/search/query/QueryPhase.java

```diff
@@ -136,6 +137,14 @@ static boolean executeInternal(SearchContext searchContext) throws QueryPhaseExe
             assert query == searcher.rewrite(query); // already rewritten
 
             final ScrollContext scrollContext = searchContext.scrollContext();
+            if (searchContext.sort() != null && searchContext.sort().sort.getSort().length == 1) {
+                // We must disable the sort optimization with point for search_after or scroll requests until LUCENE-10119 is integrated.
```

Comment:
Can you add a TODO to make it clear that an action is needed

Reply:
added in https://github.com/elastic/elasticsearch/pull/78230/commits/4cdc73a6b6003fbf7277cee239b1f35b58839c44